scripts/ci-e2e.sh

```diff
@@ -33,14 +33,14 @@ source "${REPO_ROOT}/scripts/environment.sh"
 # Clone dev-env repo
 sudo mkdir -p ${WORKING_DIR}
 sudo chown "${USER}":"${USER}" ${WORKING_DIR}
-M3_DEV_ENV_REPO="https://github.com/metal3-io/metal3-dev-env.git"
-M3_DEV_ENV_BRANCH=main
+M3_DEV_ENV_REPO="https://github.com/Nordix/metal3-dev-env.git"
+M3_DEV_ENV_BRANCH=update-capi-v1beta2/adil
 M3_DEV_ENV_PATH="${M3_DEV_ENV_PATH:-${WORKING_DIR}/metal3-dev-env}"
 clone_repo "${M3_DEV_ENV_REPO}" "${M3_DEV_ENV_BRANCH}" "${M3_DEV_ENV_PATH}"
 
 # Config devenv
 cat <<-EOF >"${M3_DEV_ENV_PATH}/config_${USER}.sh"
-export CAPI_VERSION=${CAPI_VERSION:-"v1beta1"}
+export CAPI_VERSION="v1beta2"
```

Comment:
We are hardcoding here which means we dont expect to run any v1beta1 test on main anymore?